### PR TITLE
Created and added README-AL (in Albanian)

### PR DESCRIPTION
**Pull request by tanTarantino**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.

### CODE REVIEW

File: README-AL.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        <li><a href="./README-FA.md"> فارسی </a></li>
+        <li><a href="./README-UR.md">اردو </a></li>
+        <li><a href="./README-AR.md">اللغة العربية </a></li>
+        <li><a href="./README-TA.md">தமிழ்</a></li>
+        <li><a href="./README-NE.md">नेपाली</a></li>
+    </ul>
+</details>
+<!-- Do not translate this -->
+
+# Mirë se vini kontribues fillestarë te Burimit te hapur (Open source)!
```
Comment (XhonPelushi) [inline README-AL.md:35]: # Mirë se vini kontribues fillestarë te Burimit t**ë** hapur (Open source)!

Comment (tanTarantino) [reply README-AL.md:35]: Nice catch! Thanks

**Review by tanTarantino — COMMENTED** [review]
Fixed typo by @XhonPelushi

**Review by tanTarantino — COMMENTED** [review]
fixed typo by @XhonPelushi before merging

**Pull request closed without merging**